mne/viz/topomap.py

```diff
@@ -591,6 +591,12 @@ def __init__(self, pos, extrapolate, origin, radii, border):
         assert pos.ndim == 2 and pos.shape[1] == 2, pos.shape
         _validate_type(border, ('numeric', str), 'border')
 
+        # check that border, if string, is correct
+        if isinstance(border, str):
+            if border != 'mean':
+                msg = 'border must be numeric or "mean", got {!r}'
+                raise ValueError(msg.format(border))
```

Comment:
```suggestion
            _check_option('border', border, ('mean',), extra='when a string')
```

Reply:
oh, I was wondering if I can do this with `_check_option`, thanks!